iMX8Pkg/iMX8CommonDsc.inc

```diff
@@ -389,8 +437,8 @@
   #                              // significantly impact boot performance
   #  DEBUG_ERROR     0x80000000  // Error
 #  gEfiMdePkgTokenSpaceGuid.PcdDebugPrintErrorLevel|0x8012004F
-  gEfiMdePkgTokenSpaceGuid.PcdDebugPrintErrorLevel|0x80080206
-
+  gEfiMdePkgTokenSpaceGuid.PcdDebugPrintErrorLevel|0x80080246
+  gEfiMdePkgTokenSpaceGuid.PcdFixedDebugPrintErrorLevel|0x80080246
```

Comment:
It isn't obvious, but PcdFixedDebugPrintErrorLevel masks any bits outside of this range for PcdDebugPrintErrorLevel, please either don't set this, or put a comment what this definition actually does.